test/bigtest/tests/settings-test.js

```diff
@@ -0,0 +1,21 @@
+import { beforeEach, describe, it } from '@bigtest/mocha';
```

Comment:
It is a good idea to place each fn on the new line.

Reply:
Fixed